code-challenges.test.js

```diff
@@ -63,32 +140,133 @@ var randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction
 var mixedDataArray1 = [true, 8, "hello", -8, null, 0, 46, 59, 107, "hey!"]
 // Expected output: [-8, 0, 8, 46, 59, 107]
 
-var mixedDataArray1 = [3, "yo!", 94, -9, false, 0, 18, "hola!"]
+var mixedDataArray2 = [3, "yo!", 94, -9, false, 0, 18, "hola!"]
 // Expected output: [-9, 0, 3, 18, 94]
 
+describe('when numbersSorted is called ', () => {
+  it('returns a sorted array of only numbers going smallest to largest', () => {
+    //Arrange 
+    const mixedDataArray = [true, 8, "hello", -8, null, 0, 46, 59, 107, "hey!"]
+
+    //Act
+    const actualResult = numbersSorted(mixedDataArray)
+
+    //Assert
+    expect(actualResult).toEqual([-8, 0, 8, 46, 59, 107])
+  })
 
+  it('returns a sorted array of only numbers going smallest to largest', () => {
+    //Arrange 
+    const mixedDataArray = [3, "yo!", 94, -9, false, 0, 18, "hola!"]
 
+    //Act
+    const actualResult = numbersSorted(mixedDataArray)
+
+    //Assert
+    expect(actualResult).toEqual([-9, 0, 3, 18, 94])
+  })
+})
 
 
 // b) Create the function that makes the test pass.
 
+const numbersSorted = (array) => {
+  return array.filter(value => {
+    return typeof value === 'number'
+  }).sort((a, b) => {return a - b})
```

Comment:
```suggestion
  }).sort((a, b) =>  a - b)
```
You can use an implicit return if you're looking to simplify a bit more